x/stake/endblock.go

```diff
@@ -252,3 +266,84 @@ func handleMatureUnbondingDelegations(k keeper.Keeper, ctx sdk.Context) ([]types
 
 	return completed, events
 }
+
+const (
+	maxProcessedRefundCount = 50
+)
+
+func handleRefundCrossStake(ctx sdk.Context, k keeper.Keeper) sdk.Events {
```

Comment:
The function name `handleRefundCrossStake` does not match what it does:  it not only do undelegate for cross chain stake but also for BC stake